config/puma.rb

```diff
@@ -0,0 +1,21 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+workers Integer(ENV['WEB_CONCURRENCY'] || 1)
```

Comment:
Going any higher caused the app to crash due to low memory. CF gives `512M` by default...not sure if we should consider raising it.